content/rule_book/quiz_events.md

```diff
@@ -37,10 +37,10 @@ The word "question" is significantly overloaded in the current rule book. We're
 Limited to five verses before or after the verse. On all question types, a quizzer cannot automatically be ruled out of context by a single word, even if it is a unique word. Rather, the quizzer must say a complete thought or phrase that conclusively puts them out of context.
 
 **Interchangeable**
-: Means not only not incorrect; means either are acceptable as correct.
+: Means not only not incorrect; means either may be accepted as correct.
```

Comment:
Would love discussion here. I made an interpretation--Zach made a different one.

“Jesus” and “Christ” are interchangeable and the quizzer will be called correct."

Seems like something the QM HAS to do. No "may"

Reply:
"are acceptable" is still implies can or cannot, if it is definitive I would write it as "are to be accepted"

Reply:
ah yes, very true.

Reply:
I'd like to see what Gryphon or Jeremy think about this one,

Reply:
I think the language needs to be beefed up to avoid misinterpretation. *"means either are correct"*